src/couchapps/WMStats/_attachments/index.html

```diff
@@ -4,6 +4,9 @@
   <head>
     <title>WMStats</title>
     <style type="text/css" title="currentStyle">
+        @import "css/logdb.css";
+        @import "css/kube.min.css";
```

Comment:
Thanks Valentin, hope your trip was OK.
I think that is good idea to share the code between all WMCore project.  I think the right place is under https://github.com/dmwm/WMCore/blob/master/src/html/. I am planning to move all the WMStats code there when I add REST server for WMStats, (Hopefully next release, and we can review it together.)